dev/com.ibm.ws.wsat.migration_fat.5/build.gradle

```diff
@@ -13,24 +13,22 @@
 
 // Define G:A:V coordinates of each dependency
 dependencies {
-  requiredLibs project(":com.ibm.ws.tx.embeddable"),
-               project(":com.ibm.ws.wsat.migration_fat.1"),
-               project(":com.ibm.ws.wsat.migration_fat.4"),
-               project(":com.ibm.ws.transaction.fat.util"),
-               project(':io.openliberty.org.apache.commons.logging')
+  requiredLibs project(":com.ibm.ws.wsat.migration_fat.4")
 }
 
 addRequiredLibraries.dependsOn addJakartaTransformer
 
-autoFVT.doLast {
-  copy {
-    from project(':com.ibm.ws.wsat.migration_fat.4').autoFvtDir
-    into autoFvtDir
-    include 'test-applications/**'
+task copyTestApps (dependsOn: [':com.ibm.ws.wsat.migration_fat.4:autoFVT']) {
+  doLast {
+    copy {
+      from project(':com.ibm.ws.wsat.migration_fat.4').autoFvtDir
+      into autoFvtDir
+      include 'test-applications/**'
+    }
   }
 }
 
-autoFVT.dependsOn project(':com.ibm.ws.wsat.migration_fat.4').autoFVT
+autoFVT.dependsOn copyTestApps
 
 clean.doLast {
```

Comment:
duplicated task

Reply:
i don't think so

Reply:
ok, so what is different? I see both tasks deleting the `build` dir if present.

Reply:
this one isn't using common.gradle